functions.Rmd

```diff
@@ -14,6 +14,24 @@ row_adder()
 permutation()
 ```
 
+## Anonymous functions
+
+If using an anonymous function, prefer the new lambda syntax: `\(x) {}`.
```

Comment:
nit: here we've used `{}` to convey "a function", but the rules would eliminate using `{}`, so maybe a bit misleading... I don't immediately have a good solution.